utils/build/docker/dotnet/parametric/ApmTestClient.csproj

```diff
@@ -13,6 +13,8 @@
   <ItemGroup>
     <PackageReference Include="Datadog.Trace.Bundle" Version="2.31.0" />
     <PackageReference Include="Grpc.AspNetCore" Version="2.52.0" />
+    <PackageReference Include="OpenTelemetry" Version="1.4.0" />
```

Comment:
Do we need the OpenTelemetry package for anything? I feel like we can remove it without any issues

Reply:
Correct, removed!